### DISCUSSION TITLE
DNS Records does not propagate

### DISCUSSION THREAD

**Discussion by das-ale** [vercel/community#7453] [Help]
### Summary

So a few days ago I payed a month of forwardemail, as it is recommended by Vercel, and I started the config to receive emails, adding the needed DNS Records and it worked and still works perfectly. 
A day later I wanted to be able to send messages using the @domain.com, following the tutorial on forwardemail.net I added some more DNS Records (2 TXT and 1 CNAME). It worked instantly, but after an hour, it stopped working. This was yesterday night, and now that I'm checking the DNS Records in online checkers, those TXT and CNAME don't appear. So I don't know why it worked, but now it doesn't even show in dns checkers.
Do I need to do something more for the DNS changes to be applied?
Btw, the domain is from other company but I changed the nameservers to Vercel ones and that works nice and I'm on free plan

### Steps to Reproduce

Change DNS Record

**Comment by amyegan** [comment]
Hey @das-ale. Can you tell me the domain name and some more about the DNS records needed?

[a comment by an automated account is not shown]
